app/lib/config/config.go

```diff
@@ -33,6 +33,8 @@ type CothoritydConfig struct {
 	Public  string
 	Private string
 	Address network.Address
+	TLSCert string
+	TLSKey  string
```

Comment:
these are the PEM-encoded data, so string makes more sense here. 

I could also store them as `[]byte` and add a `PEM`-method and a NewTLSKeyFromPEM`...  But that seems to be overkill to me. Even though I always have to encode/decode in PEM.